.env.example

```diff
@@ -1,120 +1,22 @@
-# Required by Postgre container
-POSTGRE_DB_PATH="postgre/db/path"
+ASSETS_ROCKS_DB_PATH="path/to/assets/db"
+ASSETS_ROCKS_DB_SECONDARY_PATH="path/to/assets_secondary/db"
```

Comment:
this one will be per service most likely. As it's secondary. Sometimes we could use temp dir for it or if we expect it to grow during the runtime.